### ISSUE TITLE
Unable to find s282_brake_shoes.obj

### ISSUE THREAD

**Issue by JPEWdev** [pjschinske/dv-rearranged-S282#1]
I tried to use this mod, but it fails to load every time claiming that it cannot find `assets/s282_brake_shoes.obj`

**Comment by pjschinske** [comment]
What OS are you using? It currently doesn't work on Linux or on the Steam deck. But a minority of windows users have it too which I don't understand.

**Comment by JPEWdev** [comment]
Linux. If there's any debugging logs that would be helpful, let me know.

**Comment by pjschinske** [comment]
Nah, it's a bug I've seen from other users as well. I'm packaging this mod with a third-party program ([this fork of AssetStudio](https://github.com/aelurum/AssetStudio)) that extracts some meshes from the game files. But AssetStudio doesn't seem to run correctly under Wine. I've also tried packaging it with the native linux version of AssetStudio and it still didn't work; I think there were some issues with accessing the right .NET runtime.

I've got some ideas on how to fix it but they might take a while unfortunately. I don't have a Linux computer capable of running Derail Valley so debugging this issue is a bit tricky.

**Comment by pjschinske** [comment]
Hey,
Someone found a solution for this, here's a walkthrough they filmed on how to do it. Does this work for you?
https://www.youtube.com/watch?v=_ABcLrGCUl8

**Comment by JPEWdev** [comment]
Ya, that fixed it. Thanks for the tip
